blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedIngestionHandler.java

```diff
@@ -80,25 +80,25 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
             List<Boolean> persisteds = futures.get(timeout.getValue(), timeout.getUnit());
             for (Boolean persisted : persisteds) {
                 if (!persisted) {
-                    DefaultHandler.sendResponse(ctx, request, null, HttpResponseStatus.INTERNAL_SERVER_ERROR);
+                    DefaultHandler.sendResponse(ctx, request, null, HttpResponseStatus.INTERNAL_SERVER_ERROR, null);
```

Comment:
I am not fond of adding a new parameter argument to an existing functions and modifying any current code that needs it to have it call by passing "null".

May I recommend you create a wrapper method also called sendResponse() in the DefaultHandler class, only needing the original parameters, and have that called your modified method with the null as the headers parameter?

Then you can remove all these modification to the endResponse() calls and not have to pass in a "null" when it's not actually needed.